src/github.ts

```diff
@@ -124,5 +124,5 @@ export function getDownloadUrlByTag (version: string, forAPICall: boolean = fals
   version = version.toLowerCase()
   return (version === '' || version === 'master' || version === 'latest')
     ? `${GITHUB_COM_SERVER_URL}/SAP/jenkins-library/releases/latest`
-    : `${GITHUB_COM_SERVER_URL}/SAP/jenkins-library/releases/${forAPICall ? 'tags' : 'tag'}/${version}`
+    : `${forAPICall ? `${GITHUB_COM_API_URL}/repos` : GITHUB_COM_SERVER_URL}/SAP/jenkins-library/releases/${forAPICall ? 'tags' : 'tag'}/${version}`
```

Comment:
this will cause errors because all API requests must be authorized, but most of the pipelines don't have github.com token configured.

Reply:
This is just a helper function, if someone needs a download Url for API call they need to call this function with `forAPICall: true`. We dont have such call for now. It's responsibility of the caller (in the future) to make sure token is provided.